scripts/run-integration-tests.sh

```diff
@@ -151,10 +154,21 @@ mkdir -p "$TEST_CLUSTER_DIR"
 mkdir -p "$TEST_CONFIG_DIR"
 
 START=$SECONDS
-if [[ "$PROVISION" == true && "$RUN_KOPS_TEST" == true ]]; then
-    up-kops-cluster
-elif [[ "$PROVISION" == true ]]; then
-    up-test-cluster
+if [[ "$PROVISION" == true ]]; then
+    START=$SECONDS
+    if [[ "$RUN_BOTTLEROCKET_TEST" == true ]]; then
+        echo "TESTING downloading eksctl"
+        curl --silent --location "https://github.com/weaveworks/eksctl/releases/latest/download/eksctl_$(uname -s)_amd64.tar.gz" | tar xz -C /tmp
+        sudo mv -v /tmp/eksctl /usr/local/bin
```

Comment:
Please place the above code into an `ensure_eksctl` Bash function (similar to `ensure_ecr_repo` and `ensure_aws_k8s_tester` functions.